package.json

```diff
@@ -59,9 +66,11 @@
       }
     ],
     "directories": {
-      "output": "release",
-      "app": "source"
+      "output": "release"
     },
+    "files": [
+      "!**/node_modules/mermaid${/*}"
```

Comment:
This line is responsible for excluding the mermaid-js related files from the released version.

Eventually the goal should be to exclude the whole `node_modules` directory.